src/main.cpp

```diff
@@ -1,11 +1,12 @@
 #include <iostream>
 #include "Kregielnia.hpp"
-#include "Helper.hpp"
+#include "TestFileSupport.hpp"
 
 int main()
 {
-    Helper::CreateTestFiles();
-    Helper::RemoveTestFiles();
+    const std::string inputfiles_directory = "test_files";
+    TestFileSupport::CreateTestFiles(inputfiles_directory);
```

Comment:
The name suggests that those functions should be used only in tests.

Reply:
I made refactor for this functions and currently this function are used only in tests.